metis-indexing/src/main/java/eu/europeana/indexing/mongo/property/LicenseUpdater.java

```diff
@@ -0,0 +1,50 @@
+package eu.europeana.indexing.mongo.property;
+
+import org.apache.commons.lang.StringUtils;
+import org.mongodb.morphia.query.Query;
+import org.mongodb.morphia.query.UpdateOperations;
+import eu.europeana.corelib.solr.entity.LicenseImpl;
+import eu.europeana.corelib.storage.MongoServer;
+
+/**
+ * Field updater for instances of {@link LicenseImpl}.
+ */
+public class LicenseUpdater implements PropertyMongoUpdater<LicenseImpl> {
+
+  @Override
+  public LicenseImpl update(LicenseImpl mongoEntity, LicenseImpl newEntity,
+      MongoServer mongoServer) {
+    Query<LicenseImpl> updateQuery = mongoServer.getDatastore().createQuery(LicenseImpl.class)
+        .field("about").equal(mongoEntity.getAbout());
+    UpdateOperations<LicenseImpl> ops =
+        mongoServer.getDatastore().createUpdateOperations(LicenseImpl.class);
+    final UpdateTrigger updateTrigger = new UpdateTrigger();
+    if (mongoEntity.getCcDeprecatedOn() != newEntity.getCcDeprecatedOn()) {
```

Comment:
Can we check this one? Is the Date value to be checked actually, instead of the Object?